mixer/adapter/svcctrl/expiringLRUCache.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Istio Authors
```

Comment:
This code has the substantial issue that if an entry is never queried, it will remain in the cache indefinitely and consume memory forever. There needs to be goroutine to forcibly evict old entries.

Reply:
The cache has a size limit, means memory consumption is bounded.